services/prometheus/scrape_configs.go

```diff
@@ -474,3 +477,168 @@ func scrapeConfigsForExternalExporter(s *models.MetricsResolutions, params *scra
 
 	return []*config.ScrapeConfig{cfg}, nil
 }
+
+// PopulateScrapeConfigs populates scrape configs for basic running service and agents.
+func PopulateScrapeConfigs(cfg *config.Config, l *logrus.Entry, q *reform.Querier, s *models.MetricsResolutions) error {
```

Comment:
Is it a good idea to make it public? Maybe it's better to create separate structure which will handle all scraper_configs and pass it, so victoriaMetrics packaged will not use prometheus package. or move this file somewhere else.
It should help us with removing prometheus once victoria will be ready.

Reply:
I think, that removing prometheus will require some work anyway and copy paste code isn't good. For now we can reuse it.